src/index.js

```diff
@@ -0,0 +1,142 @@
+const state = {
+  tempCounter: 72,
+  lat: '49.7818607',
+  lon: '20.9440505',
+  city: 'Kąśna Dolna, Poland'
+};
+
+const increaseTemp = (event) => {
+  state.tempCounter += 1;
+  const tempCounterContainer = document.getElementById('tempNumber');
+  tempCounterContainer.textContent = state.tempCounter + '°';
+  let tempCircle = document.getElementById('circle');
+  tempCircle = displayColorAndGarden(event);
+};
+
+const decreaseTemp = (event) => {
+  state.tempCounter -= 1;
+  const tempCounterContainer = document.querySelector('#tempNumber');
+  tempCounterContainer.textContent = state.tempCounter +'°';
+  let tempCircle = document.getElementById('circle');
+  tempCircle = displayColorAndGarden(event);
+};
+
+const displayColorAndGarden = (event) => {
+  let tempCircleColor = document.getElementById('tempNumber');
+  let gardenContainer = document.getElementById('earthGarden')
+  if (state.tempCounter >= 80) {
+    tempCircleColor.style.color = 'red';
+    gardenContainer.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (state.tempCounter <= 79 && state.tempCounter >= 70) {
+    tempCircleColor.style.color = 'orange';
+    gardenContainer.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (state.tempCounter <= 69 && state.tempCounter >= 60) {
+    tempCircleColor.style.color = 'yellow';
+    gardenContainer.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (state.tempCounter <= 59 && state.tempCounter >= 50) {
+    tempCircleColor.style.color = 'aquamarine';
+    gardenContainer.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (state.tempCounter <= 49 && state.tempCounter >= 30) {
+    tempCircleColor.style.color = 'teal';
+    gardenContainer.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
```

Comment:
I like that you combined the color and garden data in this method.

Check out the logic in this method, see how similar the branches are of the conditional statement. We look for whether the temperature is within a range, and pick a color/garden accordingly. What if we had a list of records that we could iterate through to find the values. We could set up something like:


```javascript
const TEMP_COLORS_GARDENS = [ 
  { upperBound: 49, color: "teal", garden: '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲'}, 
  { upperBound: 59, color: "green", garden: '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲'}, 
  { upperBound: 69, color: "yellow", garden: '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃'}, 
  { upperBound: 79, color: "orange", garden: '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷'}, 
  { upperBound: 80, color: "red", garden: '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂'}
];
```
Then your method would iterate through TEMP_COLORS_GARDENTS and find the first record that has an upper bound higher than `state.tempCounter`, then use it as the source of picking the color which would help make the function a bit more concise.

This could accommodate a scenario where you might have even more color and garden combinations in the future, which would prevent your conditional statement from being a really long block of if/elif/elif and so on